helpers/helpers.go

```diff
@@ -29,3 +29,13 @@ func GetAPIRouterVersion(rawurl string) (string, error) {
 	}
 	return apiRouterURL.Path, nil
 }
+
+// StringInSlice will check if a string is in a slice and return a corresponding boolean value
+func StringInSlice(str string, slice []string) bool {
```

Comment:
May I suggest also returning the `index`? 
```
func StringInSlice(str string, slice []string) (int, bool) {}
```
As this is a utility function I think that would be a helpful thing to return. If its not needed the caller can just use the `_` in the assignment.